src/libs/ErrorUtils.js

```diff
@@ -88,6 +88,27 @@ function getLatestErrorField(onyxData, fieldName) {
         .value();
 }
 
+/**
+ * Get the first error in the errorFields ordering by its key
+ *
+ * @param {Object} onyxData
```

Comment:
>Document object parameters with separate lines e.g. @param {Object} parameters followed by @param {String} parameters.field.

It is missing the field `errorFields`. Pretty much similar to the JSDoc on `getLatestErrorField`.